src/jolie/router/router.ol

```diff
@@ -1,7 +1,5 @@
-
 /*
 The MIT License (MIT)
-Copyright (c) 2016 Fabrizio Montesi <famontesi@gmail.com>
```

Comment:
Not sure what happens with the MIT license if you remove the license granter, but I write here that it's OK for me for this particular modification.